specification/azurearcdata/resource-manager/Microsoft.AzureArcData/preview/2023-11-01-preview/examples/CreateOrUpdateArcSqlServerAvailabilityGroup.json

```diff
@@ -0,0 +1,259 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-1111-2222-3333-444444444444",
+    "resourceGroupName": "testrg",
+    "sqlServerInstanceName": "testSqlServer_INST1",
+    "availabilityGroupName": "testAG",
+    "api-version": "2023-11-01-preview",
+    "sqlServerAvailabilityGroupResource": {
+      "location": "southeastasia",
+      "tags": {
+        "mytag": "myval"
+      },
+      "properties": {
+        "availabilityGroupId": "00000000-1111-2222-3333-444444444444",
+        "serverName": "testSqlServer",
+        "instanceName": "INST1",
+        "collectionTimestamp": "2022-05-05T16:26:33.883Z",
+        "info": {
+          "failureConditionLevel": 3,
+          "healthCheckTimeout": 30000,
+          "automatedBackupPreferenceDescription": "SECONDARY",
```

Comment:
>SECONDARY [](http://example.com/codeflow?start=51&length=9)

Why is it all caps?

Is this really a description, or is it just an enum with values primary and secondary?

          "automatedBackupPreferenceDescription": "SECONDARY",